src/coreclr/vm/methodtablebuilder.h

```diff
@@ -783,9 +783,9 @@ class MethodTableBuilder
 
         //-----------------------------------------------------------------------------------------
         // Returns the substitution to be used in interpreting the signature.
-        const Substitution &
+        const Substitution *
         GetSubstitution() const
```

Comment:
Should we make the same change in `src/coreclr/vm/methodtablebuilder.inl`?

Reply:
Nope. That is for a different type and seems to be "correct". I'd prefer to leave this PR as simply making things correct and then we can iterate on improving the contracts all up.